dev/SplitButton/SplitButton.xaml

```diff
@@ -222,21 +218,24 @@
                                 <VisualState x:Name="SecondaryButtonSpan">
                                     <VisualState.Setters>
                                         <Setter Target="SecondaryButton.(Grid.Column)" Value="0"/>
-                                        <Setter Target="SecondaryButton.(Grid.ColumnSpan)" Value="2"/>
+                                        <Setter Target="SecondaryButton.(Grid.ColumnSpan)" Value="3"/>
                                     </VisualState.Setters>
                                 </VisualState>
                             </VisualStateGroup>
                         </VisualStateManager.VisualStateGroups>
 
                         <Grid.ColumnDefinitions>
-                            <ColumnDefinition x:Name="FirstColumn" Width="*" MinWidth="{ThemeResource SplitButtonPrimaryButtonSize}"/>
-                            <ColumnDefinition x:Name="SecondColumn" Width="{ThemeResource SplitButtonSecondaryButtonSize}"/>
+                            <ColumnDefinition x:Name="PrimaryButtonColumn" Width="*" MinWidth="{ThemeResource SplitButtonPrimaryButtonSize}"/>
+                            <ColumnDefinition x:Name="Separator" Width="1" />
+                            <ColumnDefinition x:Name="SecondaryButtonColumn" Width="{ThemeResource SplitButtonSecondaryButtonSize}"/>
                         </Grid.ColumnDefinitions>
 
-                        <Grid x:Name="PrimaryBackgroundGrid"/>
+                        <Grid x:Name="PrimaryBackgroundGrid"
+                            Background="{TemplateBinding Background}"/>
 
                         <Grid x:Name="SecondaryBackgroundGrid"
-                            Grid.Column="1"/>
+                            Background="{TemplateBinding Background}"
+                            Grid.Column="2"/>
 
                         <Grid x:Name="Border"
                             Grid.ColumnSpan="2"
```

Comment:
Shouldn't this be 3 now?

Reply:
Why 3?

Reply:
There are now 3 columns, so if there were a border added, the border should go around the entire control. If you set a BorderBrush/BorderThickness on SplitButton now, what does it look like?

Reply:
![image](https://user-images.githubusercontent.com/11960777/63284866-639b7d80-c269-11e9-9f40-93dc42b5dbdd.png)
It only wraps around the first button. I'll fix it.

Reply:
Fixed here fe1c73c